galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -1996,9 +1996,30 @@ components:
     UserData:
       type: object
       properties:
-        login_id:
+        id:
+          type: string
+          description: The ID representing the user record in CouchDB.
```

Comment:
Nitpick - might be worth replacing "CouchDB" with something like "the auth store"